src/middleware/express-error.js

```diff
@@ -14,6 +16,29 @@ const expressError = ({errorCodes, entry, monitor}) => {
   return (err, req, res, next) => {
 
     const incidentId = uuid.v4();
+    if (!(err instanceof ErrorReply)) {
+      res.reportError(
+        'InternalServerError',
+        'Internal Server Error, incidentId: ' + incidentId,
+        {incidentId}
+      );
+      debug(
+        'Error occurred handling: %s, err: %s, as JSON: %j, incidentId: %s',
+        req.url, err, err, incidentId, err.stack
+      );
+      if (monitor) {
+        err.incidentId = incidentId;
+        err.method = method;
+        err.params = req.params;
+        let payload = req.body;
+        if (cleanPayload) {
+          payload = cleanPayload(payload);
+        }
+        err.payload = req.payload;
+        monitor.reportError(err, Object.assign(tags, {method}));
```

Comment:
`tags` isn't defined here.  The second argument to `monitor.reportError` gives tags to assign to the error report, and in this case the only tag we want to add is `{method}`, so you can omit the `Object.assign` bit.